cuegui/cuegui/ServiceDialog.py

```diff
@@ -108,17 +109,15 @@ def setService(self, service):
         Update the form with data from the given service.
         """
         self.__buttons.setDisabled(False)
+        self.name.setText(service.data.name)
```

Comment:
Let's avoid the `.data` attribute here if we can, and use wrapper methods instead e.g. `service.name()`.

Reply:
```suggestion
        self.name.setText(service.name())
```